.gitlab/ci/.gitlab-ci.on-push.yml

```
@@ -259,12 +259,20 @@ xpanse-prepare-testing-bucket:
 .run_tests:
   - section_start "Run Tests"
   - |
-    if [[ -f "${ARTIFACTS_FOLDER_SERVER_TYPE}/conf.json" ]]; then	
-      cp "${ARTIFACTS_FOLDER_SERVER_TYPE}/conf.json" Tests/conf.json
+    if [[ $CI_COMMIT_BRANCH == "xsoar_8_end_to_end_tests" && $SERVER_TYPE == "XSOAR SAAS" ]]; then 
```

```suggestion
    if [[ -n "${NIGHTLY}" && $SERVER_TYPE == "XSOAR SAAS" ]]; then 
```
update it when finishing up testing